lib/migrations/20190530163257_track_typeOfTest.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('underscore');
+
+exports.up = async function(knex, Promise) {
+    await knex.schema.table('students', function(t){
+        t.string('typeOfTest').after('stage').nullable();
+    });
+
+    let students = await knex('students').select('*');
+    let promises = [];
+    
+    _.each(students, (student) => {
+        if (student.partnerId) {
+            promise = knex('students').update({
+                typeOfTest: "offlineTest"
+            })
+            promises.push(promise)
+        }else{
+            promise = knex('students').update({
```

Comment:
Here also the same.

Reply:
Also, do check the DB once to double confirm if the changes are working as expected.

Reply:
changes is working properly

Reply:
I don't think so

Reply:
now it's working properly.